qutebrowser/browser/browsertab.py

```diff
@@ -427,13 +427,20 @@ def reapply(self) -> None:
         self._set_factor_internal(self._zoom_factor)
 
 
+SelectionState = enum.Enum('SelectionState', [
+    'NONE',
+    'NORMAL',
+    'LINE',
+])
```

Comment:
We use lower-case enum member names usually (see e.g. `qutebrowser/utils/usertypes.py` for some examples).

Also, can you please use the class-based API to create enums instead? See #5336.